src/components/core/index.ts

```diff
@@ -1,7 +1,8 @@
 import { Component } from '../types.js';
 import { interactionHandler } from './interaction-handler.js';
+import { messageListener } from './message-listener.js';
 import { ready } from './ready.js';
 
 export default {
-    gatewayEvents: [ready, interactionHandler],
+    gatewayEvents: [ready, interactionHandler, messageListener],
```

Comment:
Now that we've moved it to the `qbox` component, let's make these changes in `qbox/index.ts` rather than here.

Reply:
should've moved it already.